### PR TITLE
OSDOCS-2557: Add the overview page for registry book

### PR DESCRIPTION
**Pull request by xenolinux**
JIRA ticket: https://issues.redhat.com/browse/OSDOCS-2557
@xiuwang - Kindly review the change. - LGTM'd
OCP version: 4.6, 4.7, 4.8, 4.9, 4.10
[Preview](https://deploy-preview-39257--osdocs.netlify.app/openshift-enterprise/latest/registry/index.html)

### CODE REVIEW

**Comment by xenolinux** [comment]
https://github.com/openshift/openshift-docs/pull/37516#discussion_r749878676 @bmcelvee can you please confirm?

**Comment by xiuwang** [comment]
LGTM

File: registry/index.adoc [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+= {product-title} registry overview
+include::modules/common-attributes.adoc[]
+:context: registry-overview
+
+toc::[]
+
+{product-title} can build images from your source code, deploy them, and manage
+their lifecycle. To enable this, {product-title} provides an internal,
+integrated container image registry that can be deployed in your {product-title}
+environment to locally manage images. The _Registry Overview_ contains reference information and links for registries commonly used with OpenShift. It focuses on internal Image Registry.
```
Comment (jeana-redhat) [inline registry/index.adoc:11]: Question about "The _Registry Overview_ contains reference information":
Is that referring to this same assembly? If so, perhaps "This overview contains reference information" instead?

Comment (jeana-redhat) [inline registry/index.adoc:11]:
```suggestion
environment to locally manage images. The _Registry Overview_ contains reference information and links for registries commonly used with {product-title}, with a focus on the internal image registry.
```
Though that is also a lot of instances of `{product-title}` in this paragraph. Maybe there is a way to rephrase to omit a couple.

Comment (jeana-redhat) [inline registry/index.adoc:11]: Should probably go ahead and unwrap these lines

**Review by jeana-redhat — COMMENTED** [review]
Nice work! just a few small changes suggested.

File: registry/index.adoc (hunk 1 above)
Comment (xenolinux) [reply registry/index.adoc:11]: Yes. It is referring to this same assembly. Addressed the suggestion.

Comment (xenolinux) [reply registry/index.adoc]: acked

Comment (xenolinux) [reply registry/index.adoc]: acked

**Comment by kmccarron-rh** [comment]
@xenolinux - Would you please add the Jira number prefix to the title? OSDOCS-2557:

File: registry/index.adoc [hunk 2]
```diff
@@ -0,0 +1,28 @@
+[id="registry-overview"]
+= {product-title} registry overview
+include::modules/common-attributes.adoc[]
+:context: registry-overview
+
+toc::[]
+
+{product-title} can build images from your source code, deploy them, and manage their lifecycle. It provides an internal, integrated container image registry to enable this that can be deployed in your {product-title} environment to locally manage images. This overview contains reference information and links for registries commonly used with {product-title}, with a focus on the internal image registry.
```
Comment (kmccarron-rh) [inline registry/index.adoc:8]: The phrase "to enable this" seems unnecessary. Consider removing.

Comment (xenolinux) [reply registry/index.adoc]: acked

**Pull request merged**